examples/deployment/scripts/transfer/multisig_sc.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Description:
+ * This file demonstrates the example to
+   - load a signed lsig from assets and use that lsig to validate transactions
+   - create a signed lsig using sdk and use that lsig to validate transactions
+*/
+const { transferAlgo } = require("./common");
+const { transferMicroAlgos } = require("algob");
+const algosdk = require('algosdk');
+
+async function run(runtimeEnv, deployer) {
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+  const bobAccount = deployer.accountsByName.get("bob-account");
+
+  //Get Logic Signature
+  const logicSignature = await deployer.getLogicSignature("3-gold-delegated-asc.teal", []);
+
+  /*
+    These accounts are generated by using goal (goal account new)
+    The above example demonstrates a transaction validated by a smart contract signed by a multisig account with threshold of 2
+    User saves the signed logic signature file in assets/ which is extracted and used here.
+  */
+  const kmdParams = {
+    version: 1,
+    threshold: 2,
+    addrs: [
+      'STICGNGDJKFGSSHHXALD6VBEJ5XLWL7QK5UIGV3DE2ONEXNFB6ZSOMGP6Y',
+      'U6NEV736OHP7V7QHXKZFP6QAPE5PMYJOYIPWT2VZCWIMWGJS62ZK7XLKCE',
+      'Z2SFZ7X2RAKTU4JYGUYYE7LDIE6M7NYCMUQEAIKII4UDT5KYE4IJNZAGCU',
+    ],
+  };
+  const kmdMsigAddress = algosdk.multisigAddress(kmdParams);  //Generate multi signature account hash 
+
+  //Extracting multisig from dissembled file generated by goal (goal clerk compile -D <path_to_file>)
+  const Msig = await deployer.getDelegatedMsig("3-gold-msig.multilsig");  
+
+  logicSignature.msig = {};
+  Object.assign(logicSignature.msig, Msig);
+
```

Comment:
If we are loading logicSignature from goal, we should load it completely from multisig file, 
why two different methods are used to load logicSignature ?

Reply:
Complete logic signature isn't loaded from the file. It contains the multisigned logic signature field (`Msig`) which we are appending to the logic signature generated using sdk (`object.assign(..)`) 
- Refer [https://developer.algorand.org/docs/features/asc1/modes/#logic-signatures]

Reply:
please check example provided by you , it has logic signature

https://developer.algorand.org/docs/features/asc1/stateless/walkthrough/#creating-a-multi-signature-delegated-logic-sig

Reply:
also `msig` is multi-signature
and logic is stored in `logic`
![sig](https://user-images.githubusercontent.com/26249903/96339983-a77c5380-10b5-11eb-998e-75c7b083b5c5.png)


https://developer.algorand.org/docs/features/asc1/modes/#logic-signatures

Reply:
please check below comments.

Reply:
i think i am not explaining it clear here, let's discuss it on tommorow's call

Reply:
the point of loading from multisig file is that user gets everything(that's why there is logic program in it), if this file is passed to second user
- you should not load it from two different programs
- what if user has something else in loadLsig and something different in multisig file ?
It is better to load all the information from multisig (if there is only logic code in multisig, it should be converted into logic signature)

Reply:
Updated.